README.rst

```diff
@@ -163,8 +165,3 @@ Contribution
 ------------
 
 We are always happy to have new contributions. We have `marked issues good for anyone looking to get started <https://github.com/sanic-org/sanic/issues?q=is%3Aopen+is%3Aissue+label%3Abeginner>`_, and welcome `questions on the forums <https://community.sanicframework.org/>`_. Please take a look at our `Contribution guidelines <https://github.com/sanic-org/sanic/blob/master/CONTRIBUTING.rst>`_.
-
-.. |Linode| image:: https://www.linode.com/wp-content/uploads/2021/01/Linode-Logo-Black.svg
-    :alt: Linode
-    :target: https://www.linode.com
-    :width: 200px
```

Comment:
Please do not remove this.

Reply:
I didn't remove it. I just moved it up with all the others :D

Reply:
![imagen](https://github.com/sanic-org/sanic/assets/25409753/9401ba95-86a6-4bc1-8d71-4079c887f461)